src/utils/amounts.ts

```diff
@@ -0,0 +1,14 @@
+interface FiatAmountProps {
+  amount: number;
+  token: "eth" | "dai";
+}
+
+const rates = {
+  dai: 1.02,
+  eth: 2500,
+};
+
+export const fiatAmount = ({ amount, token }: FiatAmountProps): string => {
```

Comment:
Remember that as now it's a function and not a component it should be a verb (not a name). Some options could be `convertTokenAmountToFiat` or `getFiatAmount`.

Also, take into account that you are using one object as a param, but in the `timeAgo` function you are using just params. I'll choose one approach and apply the same to all the utils. If you stick with this approach, remember the `FiatAmountProps` to `FiatAmountParams` as it makes more sense, `props` keyword should be used for React components.